README.md

```diff
@@ -15,6 +15,19 @@ This repository contains the [Urban Airship](https://www.urbanairship.com) integ
 3. Reference mParticle's integration docs below to enable the integration.
 4. If you wish to utilize Urban Airship's Push Messaging capabilities, please refer to the Push Message Considerations section below
 
+## Tag-Based Segmentation
+
+All mParticle user attributes are forwarded to Airship as [tags](https://docs.airship.com/platform/android/segmentation/) which can be used to identify and segment your audience.
+
+Most clients prefer for all tags to remain constant if set. But, a tag can be removed manually by invoking removeTag directly on the Airship SDK as shown bellow.
+
+#### Java
+```java
```

Comment:
@BrandonStalnaker i think we should switch this to kotlin or show both?

Reply:
added Kotlin. Let me know if it looks right